training/general/src/environments/acpt-pytorch-1.11-py38-cuda11.3-gpu/context/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu113-py38-torch1110:{{latest-image-tag:biweekly\.\d{6}\.\d{1}}}
+
+# Install pip dependencies
+RUN pip install 'ipykernel~=6.0' \
+		'azureml-core=={{latest-pypi-version}}' \
+		'azureml-dataset-runtime=={{latest-pypi-version}}' \
+		'azureml-defaults=={{latest-pypi-version}}' \
+		'azure-ml=={{latest-pypi-version}}' \
+		'azure-ml-component=={{latest-pypi-version}}' \
+		'azureml-mlflow=={{latest-pypi-version}}' \
+		'azureml-telemetry=={{latest-pypi-version}}' \
+        'torch-tb-profiler~=0.4.0' \
+		'py-spy==0.3.12' \
+        'debugpy~=1.6.3'
+
+RUN pip install \
+        azure-ai-ml==1.2.0 \
+        azureml-inference-server-http~=0.7.0 \
+        inference-schema~=1.4.2.1 \
+        MarkupSafe==2.0.1 \
+        regex \
+	    pybind11
+
+# Inference requirements
+COPY --from=mcr.microsoft.com/azureml/o16n-base/python-assets:20220607.v1 /artifacts /var/
+RUN /var/requirements/install_system_requirements.sh && \
+    cp /var/configuration/rsyslog.conf /etc/rsyslog.conf && \
+    cp /var/configuration/nginx.conf /etc/nginx/sites-available/app && \
+    ln -s /etc/nginx/sites-available/app /etc/nginx/sites-enabled/app && \
+    rm -f /etc/nginx/sites-enabled/default
```

Comment:
I'd prefer to have the content of additional installation here and not from a random artifact

Reply:
this is a temp solution till we get a proper inference env for ACPT.